docs/MigrationGuides.md

```diff
@@ -13,3 +13,8 @@ Please follow the appropriate guide below when **upgrading to a new major versio
 - Simplify occurrences of task type `.download(.request(destination))` to `.downloadDestination(destination)`
 - Simplify occurrences of task type `.upload(.file(url))` to `.uploadFile(url)`
 - Simplify occurrences of task type `.upload(.multipart(data))` to `.uploadMultipart(data)`
+
+### Reactive MoyaProvider  Migration
+- Replace instances of `RxMoyaProvider<Target>` with `MoyaProvider<Target>` and use the `.rx` namespace to access the RxSwift API.
+- Replace instances of `ReactiveMoyaProvider<Target>` with `MoyaProvider<Target>` and use the `.reactive` namespace to access the ReactiveSwift API.
+- Check out the pull request [Eidolon's migration to Moya 9.0.0](https://github.com/artsy/eidolon/pull/669) which covers migration from subclassing reactive providers to using composition.
```

Comment:
I would start with why you would want to check out this PR, you don't need to check it if you don't really subclass reactive provider. Like:
```
If you subclass reactive provider, check out this pull request...
```